src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/ApiController.cs

```diff
@@ -315,11 +315,10 @@ private string GetEncodedUrl(ShellSettings shellSettings, string token)
                 pathString = pathString.Add('/' + shellSettings.RequestUrlPrefix);
             }
 
-            QueryString queryString;
-
+            var queryString = String.Empty;
             if (!String.IsNullOrEmpty(token))
             {
-                queryString = QueryString.Create("token", token);
+                queryString = QueryString.Create("token", token).ToString();
             }
 
             return $"{Request.Scheme}://{hostString + pathString + queryString}";
```

Comment:
I will revert this change as it was the right usage, building all uri components, an `HostString`, `PathString` and a `QueryString`, and adding them to build a string, so that their `ToString()`, that does the right encoding for each uri component, are implicitly called.

Yes, here it would be the same as the `ToString()` of the `QueryString` returns an empty string if applied on a null `QueryString`, but i prefer to keep the pattern of adding uri components.

 https://github.com/dotnet/aspnetcore/blob/c925f99cddac0df90ed0bc4a07ecda6b054a0b02/src/Http/Http.Abstractions/src/QueryString.cs#L53-L68

Reply:
> @dafergu2
> 
> Good job! I did a little review and i will approve it.
> 
> But first can you try to use this nuspec convention?
> 
> buildMultitargeting	(4.0) .props and .targets in the buildMultitargeting folder, for cross-framework targeting
> So e.g. the following, and in other related places.
> 
> ```
>  <PackagePath>buildMultitargeting\OrchardCore.Application.Cms.Core.Targets.props</PackagePath>
> ```
> 
> https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files
> 
> **Update:** And according to the choice we will have to update the comment in the `Package.Build.props` file.

Thanks!  I'm working on testing the nupsec change now.  One question on process, there were a lot of comment change updates that seem to update style in separate commits.  Is that due to an extension/editorconfig to autocorrect?  If so, is there, can you point me to it so that I can make sure the styling fits expected guidelines?  A follow up question is, do you want me to collapse all of the related commits into a single commit to keep the history a little cleaner?  For example, all of the ones that update comments into a single commit, the ones for reverting the ApiController change back to using the struct all the way through in one commit?  Some projects want that to be done to make history neat, others don't, so just wanted to check before I made the changes.